package.json

```diff
@@ -21,6 +21,19 @@
     "node-uuid": "~1.4.7"
   },
   "devDependencies": {
-    "coffeelint": "^1.9.7"
+    "standard": "*"
+  },
+  "standard": {
```

Comment:
To match other Atom packages that use standard, this should use the following block instead:
```
"standard": {
    "env": [
      "browser",
      "node",
      "atomtest",
      "jasmine"
    ],
    "globals": [
      "atom"
    ]
  },
```
See for example https://github.com/atom/open-on-github/blob/58899b80b73f26dbe75e9938a61dfd3b8690de9e/package.json#L32-L42

Reply:
Thanks, @50Wliu. Updated in 023e95a.